README.md

```diff
@@ -8,11 +8,36 @@ Two different methods are available.
 
 Since the used methods can be very different, we will explain them individually:
 
-YOLO
-----
+## YOLO
 
-For the YOLO method it is assumed you already have a trained YOLO that can detect your objects.
-We also assume that you only need predictions for a bounding box.
+
+### Create labels for training
+We provide a script for creating YOLO labels ([createYoloLabels.py](https://github.com/bit-bots/AutoImageLabeler/blob/master/yolo/createYoloLabels.py)).
+The script currently works for the classes `ball`, `goalpost`, `robot`, `L-Intersection`, `T-Intersection`, `X-Intersection`, and the `crossbar`.
+For the intersections, we create a bounding box of 5% of the image height and 5% of the image width.
+The labels must be provided in the form of a yaml file.
+You can call the script with e.g. `python3 createYoloLabels.py /absolute/path/to/imagesets/`.
+To handle multiple imagesets, the script ignores images and yaml files in the `imagesets` folder itself and expects them in subfolders.
```

Comment:
How does ignoring help handling multiple imagesets?

Reply:
I looked over the code again and I am not sure why it was handled this way previously... A fix is proposed in #11.
Thank you for asking!